include/vcpkg/base/expected.h

```diff
@@ -208,6 +208,18 @@ namespace vcpkg
             return *m_t.get();
         }
 
+        const T& value() const&
```

Comment:
I think this is kind of backwards; I'd prefer to change error() to require a `VCPKG_LINE_INFO` as well since running those down if busted is difficult.